### PR TITLE
fix fromSelf always being truthy

### PR DESCRIPTION
**Pull request by itswillta**
Linked issue: https://github.com/zustandjs/use-zustand/issues/8.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -31,8 +31,8 @@ export function useZustand<State, Slice>(
     () => [slice, store],
   );
   useEffect(() => {
-    const unsubscribe = store.subscribe(rerender as DispatchWithoutAction);
-    (rerender as DispatchWithoutAction)();
+    const unsubscribe = store.subscribe(() => rerender(false));
+    rerender(false);
```
Comment (dai-shi) [inline src/index.ts]:
```suggestion
    const unsubscribe = store.subscribe(() => (rerender as DispatchWithoutAction)());
    (rerender as DispatchWithoutAction)();
```
is smaller in bundle.

Comment (itswillta) [reply src/index.ts]: I see. I was wondering why you weren't making it more readable.

**Review by dai-shi — COMMENTED** [review]
LGTM
Can you check the CI error please?

**Review by dai-shi — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**